Reflections/Week_2_reflection.md

```diff
@@ -0,0 +1,21 @@
+## Accomplishments
+
+The previous week has been rather slow. I spent most of my time working on building my program on windows for windows users to test my current project, however I was blocked significantly by linking issues. I also started a branch reworking the beizer curve implementation in order to reduce possible future errors.
+
+## Challenges
+
+The biggest challenge was that attempting to build the program on windows. I had attempted to build the program in a windows VM, using both MSVC and QtCreator, with many different modifications to my CMake files. Nothing that I attempted worked. I would either be unable to link correctly to the Qt libraries, or unable to properly link to the FastNose2 libraries. After about 4 concerted days of work on this, I had to give up, as I could not get it to work at all.
```

Comment:
This is a big problem, I agree with you here. However, considering the difficulty (and that your dependencies are also cross-platform), I would encourage you to continue working as you have been and put this issue on the back-burner. So long as your own code is cross-platform (which, it probably is considering the usage of QT), this is an issue that can be remedied in the future with tooling rather than code.